styles.css

```diff
@@ -1127,9 +1145,10 @@ form {
     width: 330px;
   }
 
-  .fullname-input {
+  /* .fullname-input {
     display: none;
-  }
+    visibility: hidden;
+  } */
```

Comment:
Great job team🤝

- [ ] [Optional] Kindly remove this piece of commented code from your style sheet as this is not regarded as a good practice.